ci/steps.yml

```diff
@@ -175,13 +175,33 @@ steps:
     displayName: Build accelbubble example application to test for android
 
   ##----------------------------------------------------
-  # determine Windows build system
+  # Cache Powershell Modules in $MODULES_FOLDER
+  - task: Cache@2
+    inputs:
+      key: '"pwsh modules" | "$(WINDOWS_VM_IMAGE)"'
+      path: $(MODULES_FOLDER)
+      cacheHitVar: PSModules_IsCached
+    condition: |
+        and(eq(variables['Agent.OS'], 'Windows_NT'), 
+            eq(variables['SUBCOMMAND'], 'install-qt'))
+    displayName: Cache Powershell Modules
+  # On cache miss: Download Powershell Modules to $MODULES_FOLDER
   - powershell: |
       Install-PackageProvider NuGet -Force
       Import-PackageProvider NuGet -Force
       Set-PSRepository -Name PSGallery -InstallationPolicy Trusted
-      Install-Module Pscx -AllowClobber -AllowPrerelease
-      Install-Module VSSetup -Scope CurrentUser
+      md -p $(MODULES_FOLDER)   # mkdir
+      Save-Module -Name Pscx -AllowPrerelease -Path $(MODULES_FOLDER)
+      Save-Module -Name VSSetup -Path $(MODULES_FOLDER)
```

Comment:
```suggestion
```
Maybe this part should be removed, since VSSetup should already exist in the `windows-2019` image, and will often be more up-to date than our cached copy?